react-state/my-app/src/Caption.tsx

```diff
@@ -0,0 +1,12 @@
+import { useState } from 'react';
+
+type Props = {
+  caption: string[];
```

Comment:
Since this holds more than one caption, it would be best to give it a plural name. Same with all the other components that were changed to take arrays.